addons/bestja_organization/models.py

```diff
@@ -214,13 +214,20 @@ class UserWithOrganization(models.Model):
 
     coordinated_org = fields.One2many('organization', inverse_name='coordinator')
     organizations = fields.Many2many(
-        'res.users',
+        'organization',
         relation='organization_volunteers_rel',
         column1='volunteer',
         column2='organization',
         string="Organizacje"
     )
 
+    def __init__(self, pool, cr):
+        super(UserWithOrganization, self).__init__(pool, cr)
+        cls = type(self)
+        cls.FIELDS_ACCESS_ALL += ['coordinated_org']
+        cls.FIELDS_ACCESS_PRIVILEGED += ['organizations']
+        cls.FIELDS_ACCESS_OWNER += ['organizations']
```

Comment:
I was thinking about this a lot for the past two days and end up deciding this this will be ok. I will talk to you and explain why in person, ok?